### DISCUSSION TITLE
Create new Nextjs app

### DISCUSSION THREAD

**Discussion by karim-alomar** [vercel/next.js#60626] [Help] [answered]
### Summary

**I'm trying to create a new NextJS application but I'm getting this error:**
<img width="1470" alt="Screenshot 2024-01-14 at 3 31 57 PM" src="https://github.com/vercel/next.js/assets/47941750/40d6336a-62a1-4712-ab66-c00225c4e2b5">

**Comment by icyJoseph** [comment, marked as answer]
Did you try again? Or maybe, clear your npm cache? 

```console
npm cache clean --force
```

**Reply by karim-alomar** [reply]
Thank you so much, it's done
